### DISCUSSION TITLE
I really love Open Video Downloader. Much better than Downline!

### DISCUSSION THREAD

**Discussion by selfKaiHarness** [jely2002/youtube-dl-gui#268] [General]
Title says it all. I really love using this program, why I couldn't use it before in the past? Well me being stupid, I used Downline. Their download speeds used to be fast until it slows down to a crawl due to the recent API change. 

I've now completely given up using Downline in favor of using Open Video Downloader. Keep it up to whoever is running this!

**Comment by jely2002** [comment]
Thank you for the kind words! I don't have a lot of time on my hands. But whenever possible I try to work on this!
